tests/mocks/languages.ts

```diff
@@ -0,0 +1,19 @@
+const languages = [
```

Comment:
Cannot we reuse values of `LOCALE_NAME` map inside `src/utils/locale.ts` (after exporting it) instead of keeping the mock and the actual implementation un-connected and also to keep things DRY?